tensorflow/contrib/tensor_forest/kernels/stats_ops.cc

```diff
@@ -275,10 +291,22 @@ class ProcessInputOp : public OpKernel {
     // with mutexes.
     std::unordered_map<int, std::unique_ptr<mutex>> locks;
     std::unordered_map<int32, std::vector<int>> leaf_examples;
+    std::unordered_map<int32, std::vector<int>> leaf_random_features;
```

Comment:
i know the name conforms to an obscure naming scheme (leaf_examples), but can we do better? 
leaf_feature_to_use_per_split or something like this